core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/SequencingParameters.java

```diff
@@ -71,6 +71,8 @@ public class SequencingParameters
   @Column
   private Integer movieTime;
 
+  private Integer flows;
```

Comment:
Does this need the `@Column` annotation?

Reply:
`@Column` is only really useful if it has parameters. If you want Hibernate to ignore a field, you need to tell it that explicitly (probably via `@Transient`)